Automattic/wc-calypso-bridge
# Optimize bridge file size by  lazy-loading components

### Changes proposed in this Pull Request:



Closes #962.

This PR optimizes bridge entry file size by lazy-loading payment-gateway-suggestions and marketing component.

- Add `version` prop to `window.wcCalypsoBridge`
- Append `version` query param to chunk file (Copied from https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/index.js#L27-L35)
- Remove `src/payment-gateway-suggestions/shared.scss` since we already have a global shared scss file.
- Update `placeholder` mixin style (copied from https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/internal-style-build/abstracts/_mixins.scss#L22-L35)






### How to test the changes in this Pull Request:




1. Run `npm run start` to build frontend changes.
2. Choose a WCPay non-eligible country such as Taiwan
3. Open the network tab in your browser dev tools.
4. Navigate to `WooCommerce -> Home`
5. Confirm `payment-gateway-suggestions.js` is loaded with a version query parameter.

![Screenshot 2023-05-05 at 16 44 58](https://user-images.githubusercontent.com/4344253/236414463-b3b9506e-a239-48e9-9d88-213b11124985.png)

6. Click payments task.
7. Confirm there payment task is shown and there is a free trial notice

![Screenshot 2023-05-05 at 16 49 04](https://user-images.githubusercontent.com/4344253/236415165-6c0da7de-ba6e-4d21-bc8a-2649fe863081.png)

8. Go to the Marketing page and confirm it's loaded correctly.




### Other information:




### FOR PR REVIEWER ONLY:
